cirq-core/cirq/ops/gateset.py

```diff
@@ -176,16 +220,23 @@ def _json_dict_(self) -> Dict[str, Any]:
             'name': self.name,
             'description': self.description,
             'ignore_global_phase': self._ignore_global_phase,
+            'tags_to_accept': list(self._tags_to_accept),
+            'tags_to_ignore': list(self._tags_to_ignore),
         }
 
     @classmethod
     def _from_json_dict_(
-        cls, gate, name, description, ignore_global_phase, **kwargs
+        cls, gate, name, description, ignore_global_phase, tags_to_accept, tags_to_ignore, **kwargs
```

Comment:
I think this will be a breaking change because older serializations will raise an error when trying to deserialize? 

tags_to_accept and tags_to_ignore parameters should have a default value and we should not modify the old json serialization in json serialization test files to maintain backwards compatibility. Instead, we should add new json serializations with the additional two parameters to test the updated serialization logic